lib/kafka_producer.js

```diff
@@ -64,6 +64,27 @@ function KafkaProducer(options, callback) { // eslint-disable-line
             self.batching = false;
         }
 
+        self.enableAudit = false;
+        self.auditTopicName = 'chaperone2-audit-rest-proxy-client';
+        self.auditTier = 'rest-proxy-client-nodejs';
+        self.auditDatacenter = 'unknown';
+        self.auditTimeBucketIntervalInSec = 600;
+        if ('enableAudit' in options) {
```

Comment:
Please remove these if statements and replace them with something like the following: 

self.enableAudit = false || self.enableAudit = options.enableAudit;
self.auditTopicName = 'chaperone2-audit-rest-proxy-client' || options.auditTopicName;